components/dashboard/src/experiments/client.ts

```diff
@@ -35,10 +35,9 @@ export function getExperimentsClient(): Client {
 function newProductionConfigCatClient(): Client {
     // clientKey is an identifier of our ConfigCat application. It is not a secret.
     const clientKey = "WBLaCPtkjkqKHlHedziE9g/TwAe6YyftEGPnGxVRXd0Ig";
-    const client = configcat.createClient(clientKey, {
+    const client = configcat.createClientWithLazyLoad(clientKey, {
         logger: configcat.createConsoleLogger(LogLevel.Error),
-        pollIntervalSeconds: 60 * 3, // 3 minutes
-        maxInitWaitTimeSeconds: 0,
+        cacheTimeToLiveSeconds: 60 * 1, // 1 minute
     });
```

Comment:
I'm not sure if this is desired behaviour from a UX perspective. This would block a UI element until the flags have been fetched.

What's the driver for this change?

Reply:
> What's the driver for this change?

Just read the description as well.

Reply:
> Each workspace page is loading the dashboard in iframe to show loading screen while workspace is not running. It causes a lot of requests each 3 minutes to ConfigCat.

> I'm not sure if this is desired behaviour from a UX perspective. This would block a UI element until the flags have been fetched.

We can reduce timeout time to 1500 milliseconds as we did in VS Code Desktop.

Reply:
I increased cache time to 3 mins and set timeout to 1500 milliseconds.

Reply:
Makes sense. I think I'm okay with this. Will just play around in preview.

Reply:
Also looking at the code it does not seem that it can block, since flags are compute in effect, so in worst case flags won't be applied.

Reply:
Sorry, I should've been clearer. I meant block as in not show for example the Usage UI immediately, only after it's fetched.

Reply:
We can call force update once on page load?

Reply:
> Sorry, I should've been clearer. I meant block as in not show for example the Usage UI immediately, only after it's fetched.

I don't think timing is changing. The client is created only on demand in use effect.